### PR TITLE
Add a dedicated man page for wLog

### PR DESCRIPTION
**Pull request by hardening**

### CODE REVIEW

**Comment by bmiklautz** [comment]
@freerdp-bot test.

**Comment by bmiklautz** [comment]
Hmm TestConnect timeouts quite often lately. We should have a look.

File: winpr/CMakeLists.txt [hunk 1]
```diff
@@ -164,6 +164,14 @@ if(BUILD_TESTING)
 	add_subdirectory(test)
 endif()
 
+if(WITH_MANPAGES)
+	if(OPENBSD)
+	    install(FILES wlog.1 DESTINATION man/man1)
```
Comment (akallabeth) [inline winpr/CMakeLists.txt]: @hardening Why is this case necessary? Shouldn't that be handled by CMake?

**Comment by akallabeth** [comment]
@hardening Maybe also add the perference order: Environment is overwritten by command line arguments and add the command line arguments in the man page

File: winpr/CMakeLists.txt (hunk 1 above)
Comment (hardening) [inline winpr/CMakeLists.txt]: I don't know, it was done this way for other pages in libfreerdp.

**Comment by hardening** [comment]
@akallabeth I will add the order. For command line arguments I'd prefer to put them in the freerdp man page as wLog is used can be used in other projects (you see what I mean).

**Comment by hardening** [comment]
@akallabeth adding wlog man page reference in /log-level and /log-filter forced me to do a big cleanup in man page generation.

File: winpr/CMakeLists.txt (hunk 1 above)
Comment (akallabeth) [inline winpr/CMakeLists.txt]: @bceverly Maybe you can give some hint to this line?

**Comment by hardening** [comment]
@akallabeth I have added a section stating the precedence of arguments on environment variables

**Comment by hardening** [comment]
Anybody to comment and merge this one ?

**Comment by akallabeth** [comment]
@nfedera @bmiklautz Any objections? Otherwise I'll merge this one.

**Pull request merged**